test/node/connection-handshake.js

```diff
@@ -126,3 +126,17 @@ test.test('must not accept handshakes on a client', (test) => {
   const connection = new jstp.Connection(transport, null, {});
   transport.emitPacket(handshake);
 });
+
+test.test(
```

Comment:
@lundibundi I've added check that `HANDSHAKE_TIMEOUT` occurs, but there is no reason in checking id of a connection, because there is only one connection.